tests/test_experiment_server.py

```diff
@@ -836,6 +836,30 @@ def test_post_participant_calls_original_route(self, a, db_session, webapp):
                 entry_information={"additional_stuff": "1"},
             )
 
+    def test_post_participant_removes_fingerprint(self, a, db_session, webapp):
+        with mock.patch(
+            "dallinger.experiment_server.experiment_server.create_participant"
+        ) as create_participant:
+            create_participant.side_effect = lambda *args, **kw: "Result"
```

Comment:
Any reason not to use `create_participant.return_value = "Result"`?

Reply:
I just copied and pasted this from the test above 🙈. I think in this case `return_value` would probably be fine.